hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/lock/OBSKeyPathLockStrategy.java

```diff
@@ -31,6 +33,9 @@
  */
 public class OBSKeyPathLockStrategy implements OzoneLockStrategy {
 
+  private OMHashCodeGenerator omHashCodeGenerator =
+      new DefaultOMHashCodeGeneratorImpl();
```

Comment:
Add a TODO saying that, "TODO: need to make this pluggable and allows users to configure the preferred hashcode generation mechanism."

Reply:
I have added the TODO comment, thanks!